plugin_export.go

```diff
@@ -5,12 +5,41 @@ package vst2
 //#include "include/vst.h"
 import "C"
 import (
+	"reflect"
+	"runtime"
+	"syscall"
 	"unsafe"
 )
 
+//fix vst host crash in windows, when plugin gets unloaded by pinning dll
+func preventDllFromUnload() {
```

Comment:
Couple of steps you need to take to make it work for non-windows OS:

1. This whole function should go to new `plugin_export_windows.go` file; 
2. You can rename it to `loadHook` as it should be defined for all systems;
3. You'll also need to add new file `plugin_export_notwin.go` with build tag `// +build !windows` also declare `loadHook` there, but it can be empty for now. 

This should fix the problem and CI will be green.